web-ng/src/app/components/card-view-project/card-view-project.component.html

```diff
@@ -0,0 +1,29 @@
+<div fxLayout="column">
+    <gnd-header-layout>
+    </gnd-header-layout>
+  
+    <h1 fxFlex style="padding: 16px;"> Recent Projects</h1>
+    <div fxFlex class="container-projects">
+          <mat-grid-list [cols]="breakpoint" rowHeight="2:1" (window:resize)="onResize($event)">
+            <mat-grid-tile>
+              <mat-card  (click)="onNewProject()" class="project-card">
+                  <mat-card-header>
+                    <mat-card-title>
+                    <button mat-icon-button color="primary">
+                      <mat-icon>home</mat-icon>
+                    </button>
+                  </mat-card-title>
+                  </mat-card-header>
+              </mat-card>
+            </mat-grid-tile>
+            <mat-grid-tile *ngFor="let project of projects; let i = index" >
+              <mat-card (click)="onProjectClicked(i)"  class="project-card">
```

Comment:
Instead of passing the index to the event handler, you can just pass the project object itself here.